index.html

```diff
@@ -89,7 +91,9 @@ <h3 class="speaker-main">Featured Speakers</h3>
     <div class="speaker-list show-more">
         
     </div>
-    <button type="submit" class="more">More &nbsp; <i class="fa-light fa-angle-down"  ></i></button>
+    <div class="more">
+        More <i class="fa fa-angle-down"></i>
+    </div>
```

Comment:
- [Optional] Please use an indentation of two spaces only in all your documents for readability. This is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and also a convention.